changelogs/unreleased/gh-5295-split-brain-detection.md

```diff
@@ -0,0 +1,9 @@
+## bugfix/replication
+
+* Fixed a possible split-brain when old synchro queue owner might finalize the
+  transactions in presence of a new synchro queue owner (gh-5295).
+
```

Comment:
This is not split-brain. I wonder why you call it this way. Split brain is a specific case of network partitiong when neither side has a majority. It's impossible to detect. So you can't really have a split brain detection.